src/css/math.less

```diff
@@ -298,6 +298,21 @@
     }
   }
 
+  .mq-limit {
+    text-align: center;
+
+    span  {
+      display: block;
```

Comment:
@laughinghan below added `span > span` rule to fix this issue. We had it in our codebase but on a separate commit, which I missed